<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipientMainWindow</class>
 <widget class="QDialog" name="RecipientMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>651</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bradley Hand ITC</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#RecipientMainWindow
{
background-color: rgb(224, 241, 255);
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>241</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox
{
background-color:#006ba1;
/*background-color: #007cba;*/
border-radius: 10px;
margin-right: 3px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_3{
color: white;
}</string>
    </property>
    <property name="text">
     <string>Search for a specific Blood</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_4{
color:white;
}</string>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#searchButton
{
color:black;
background-color: white ;
border-radius: 10px;

}

#searchButton:hover{
background-color: rgb(207, 207, 207);
}</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>280</y>
     <width>281</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox_2
{
background-color:#005a87; 
border-radius: 10px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_5{
color:white;

}</string>
    </property>
    <property name="text">
     <string>Request Blood</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_6{

color:white;
}</string>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_7{

color:white;
}</string>
    </property>
    <property name="text">
     <string>Quantity</string>
    </property>
   </widget>
   <widget class="QComboBox" name="requestCombobox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>101</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>A+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>O+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>O-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AB+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AB-</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="requestButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#requestButton
{
color:black;
background-color: white ;
border-radius: 10px;

}

#requestButton:hover{
background-color: rgb(207, 207, 207);
}</string>
    </property>
    <property name="text">
     <string>Request</string>
    </property>
   </widget>
   <widget class="QComboBox" name="quantity">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>350</y>
     <width>261</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox_3
{
background-color: #006ba1;
border-radius: 10px;
margin-left: 3px;
}
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label{
color:white;
}</string>
    </property>
    <property name="text">
     <string>Show All Avalibale Blood</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_2
{
color:white;
}</string>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showAllBlood">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#showAllBlood{
color:black;
background-color: white ;
border-radius: 10px;
}

#showAllBlood:hover{
background-color: rgb(207, 207, 207);
}</string>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="updateRecipient">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>30</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#updateRecipient
{
color:rgb(255, 243, 243);
background-color: #006ba1;
border-radius: 10px;
}

#updateRecipient:hover{

background-color:  #007cba ;
}</string>
   </property>
   <property name="text">
    <string>Update Account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteRecipient">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>30</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#deleteRecipient{
color:rgb(255, 243, 243);
background-color:  #007cba ;
border-radius: 10px;

}

#deleteRecipient:hover{
background-color: #006ba1;
}</string>
   </property>
   <property name="text">
    <string>Delete Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Magneto</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_8{

	color: rgb(0, 107, 161);
}</string>
   </property>
   <property name="text">
    <string>ATBR3LY</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_9{

	color: rgb(0, 107, 161);
}</string>
   </property>
   <property name="text">
    <string>Welcome, Recipient</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>580</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#BackButton{
color:rgb(255, 243, 243);
background-color:  #005a87 ;
border-radius: 10px;

}

#BackButton:hover{
background-color: #006ba1;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
